src/Lighthouses.java

```diff
@@ -0,0 +1,170 @@
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+/* og�lnie je�li user wybierze pole z latarni� to nie mo�na mu na to pozwoli�, czyli je�li wstawi statek
+ * w pole z latarni� musi wyst�pi� wyj�tek (je�li pole do wstawienia statku b�dzie r�ne od '_'')
+ * trzeba b�dzie parstowa� zapisane pola na int �eby sprawdzi� ile statk�w o�wietl�
+ * trzeba zabroni� userowi wpisania innego znaku ni� int*/
+public class Lighthouses {
+
+    char board[][] = new char[12][12];
+    Scanner key = new Scanner(System.in);
+
+    public static void main( String[] args ) {
+        Scanner key = new Scanner(System.in);
+        Lighthouses lighthouses = new Lighthouses();
+        lighthouses.setBoard();
+        System.out.println("Tak wygl�da tablica do kt�rej trzeba wstawi� statki:");
```

Comment:
Nie mów graczowi o tablicy... dla niego to raczej plansza ze statkami. Plansza gry. nie tablica.